packages/fx-core/src/plugins/resource/apim/utils/telemetry.ts

```diff
@@ -22,14 +22,12 @@ class TelemetryPropertyName {
 export class Telemetry {
   public static sendLifeCycleEvent(
     telemetryReporter: TelemetryReporter | undefined,
-    envInfo: EnvInfo,
     lifeCycle: PluginLifeCycle,
     status: OperationStatus,
     error?: UserError | SystemError
   ): void {
-    const solutionConfig = new SolutionConfig(envInfo);
-    const properties = solutionConfig.remoteTeamsAppId
-      ? { appid: solutionConfig.remoteTeamsAppId }
+    const properties = solutionGlobalVars.TeamsAppId
```

Comment:
Is solutionGlobalVars available in API v2?

Reply:
Yes